samples/drivers/lora_recv/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.13.1)
```

Comment:
same comments to receiver as what I wrote for sender

Reply:
Okay